rest/model/artifact.go

```diff
@@ -11,10 +11,14 @@ import (
 )
 
 type APIFile struct {
-	Name           *string `json:"name"`
-	Link           *string `json:"url"`
-	URLParsley     *string `json:"url_parsley"`
-	Visibility     *string `json:"visibility"`
+	// Human-readable name of the file
```

Comment:
Some of the docs have periods at the end of the doc, and some don't. Should they be consistent?

Reply:
Future ticket thinking about what to do about doc formatting SGTM.